apps/omg_eth/lib/omg_eth/release_tasks/set_ethereum_client.ex

```diff
@@ -52,6 +53,15 @@ defmodule OMG.Eth.ReleaseTasks.SetEthereumClient do
     rpc_client_type
   end
 
+  defp get_eth_call_from_address() do
+    eth_call_from_address =
+      validate_address(get_env("ETH_CALL_FROM_ADDRESS"), Application.get_env(@app, :eth_call_from_address))
```

Comment:
yeah, so... shouldn't this simply be the authority address? why does this have to be a different address?

Reply:
> yeah, so... shouldn't this simply be the authority address? why does this have to be a different address?

I was thinking about that. I think we do not have access to authority address in watcher and I wanted to keep things consistent. I think it's the old mentality of having child-chain and watcher together. But you are right. It doesn't make sense, we can use authority address.

Reply:
or simply use the plasma_framework contract address everywhere!